src_web/kamaho-shokusu/tests/TestCase/Service/ActualMealManagementServiceTest.php

```
@@ -0,0 +1,173 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Test\TestCase\Service;
+
+use App\Service\ActualMealManagementService;
+use Cake\ORM\TableRegistry;
+use Cake\TestSuite\TestCase;
+
+class ActualMealManagementServiceTest extends TestCase
```

PHPStan (2.2.2)

[error] Class App\Test\TestCase\Service\ActualMealManagementServiceTest extends unknown class Cake\TestSuite\TestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)